app/src/RoomClient.js

```diff
@@ -2266,15 +2268,19 @@ export default class RoomClient
 
 		let newPreferredTemporalLayer = consumer.temporalLayers - 1;
 
-		if (newPreferredSpatialLayer === 0)
+		if (newPreferredSpatialLayer === 0 && newPreferredTemporalLayer > 0)
 		{
-			const lowestLevelArea = width * height / (resolutionScalings[0] ** 2);
+			const lowestLevelWidth = width / resolutionScalings[0];
+			const lowestLevelHeight = height / resolutionScalings[0];
 
-			if (availableArea < lowestLevelArea * 0.5 && newPreferredTemporalLayer > 0)
```

Comment:
@Astagor we could trigger the temporal layer scalability when the number of active video consumers is greater than a configured threshold (https://github.com/edumeet/edumeet/pull/657)

Reply:
This was what I have been doing originally in my PR, but this means that we have to make calls from RoomClient and Spotlights, as this is not more related to the width and height. As for example I have 6 active consumers the threshold is 5. I pause 1 consumer, so a higher resolution should be shown, but your code now would not trigger it , as the size of the videos did not change. So this has to be monitored actively in methods like pauseConsumer, resumeConsumer, updateSpotlights, etc.

Reply:
Have a look at #657 
@vpalmisano

Reply:
Yes I've seen it. We need to track when the number of active video consumers changes and report it to the adaptation function.